### ISSUE TITLE
Trigger does not work with Home Assistant v0.109

### ISSUE THREAD

**Issue by technixp** [mattsaxon/sonoff-lan-mode-homeassistant#34]
Devices are visible, device status is updating, but I cannot trigger them

Debug output on trigger:
```
2020-04-29 22:13:57 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event call_service[L]: domain=switch, service=turn_off, service_data=entity_id=switch.lights_aqua_main>

2020-04-29 22:13:57 DEBUG (MainThread) [custom_components.sonoff_lan_mode_r3.switch] HassSonoffSwitch returning _available: True
```

**Comment by oopiicaa** [comment]
It broke the integration for me too after updating to 0.109. The same behaviour as @technixp described. State of the switch updates in HA if it's changed in the eWeLink app, but cannot change the state via HA.

**Comment by Bluhme1** [comment]
Same here

**Comment by mattsaxon** [comment]
Fixed with version 1.1.4.1

Was a rename of a couple of functions, that have presumably been deprecated in 109 as it moves towards async functions
